Makefile

```diff
@@ -13,13 +13,15 @@ SKIPPER_PARAMS ?= -i
 
 # assisted-service
 SERVICE_BRANCH := $(or $(SERVICE_BRANCH), "master")
+SERVICE_BASE_REF := $(or $(SERVICE_BASE_REF), "master")
```

Comment:
add doc on readme for those new vars

Reply:
Made on a different PR https://github.com/openshift/assisted-test-infra/pull/955